src/app/core/store/organizations/organizations.effects.ts

```diff
@@ -85,9 +90,13 @@ export class OrganizationsEffects {
 
   @Effect()
   public delete$: Observable<Action> = this.actions$.ofType<OrganizationsAction.Delete>(OrganizationsActionType.DELETE).pipe(
-    switchMap(action => this.organizationService.deleteOrganization(action.payload.organizationCode).pipe(
-      map(() => action)
-    )),
+    withLatestFrom(this.store$),
```

Comment:
The same comment applies here. We should use selectors as much as possible otherwise any refactoring of the store structure will need many changes across the whole application.

I can see it in the `ProjectEffects` as well.